.github/workflows/test.yml

```
@@ -8,6 +8,24 @@ env:
   BUNDLE_WITHOUT: release
 
 jobs:
+  lint:
+    runs-on: ubuntu-latest
+    strategy:
+      fail-fast: true
+      matrix:
+        include:
+          - ruby: "2.5"
+    name: Lint - Ruby ${{ matrix.ruby }}
+    steps:
+      - uses: actions/checkout@v2
```

v2 is deprecated, please switch to v3
```suggestion
      - uses: actions/checkout@v3
```